identify/README.md

```diff
@@ -3,7 +3,8 @@
 The identify protocol is used to exchange basic information with other peers
 in the network, including addresses, public keys, and capabilities.
 
-There are two variations of the identify protocol, `identify` and `identify/push`.
+There are three variations of the identify protocol, `identify`,
+`identify/push`, and `identify/delta`.
```

Comment:
we should probably specify the exact protocol IDs in a reference table here, even if we repeat them in prose below.